src/index.html

```diff
@@ -5,9 +5,61 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Apple iMac 27&quot;"
+        class="card__img">
+
+      <p class="card__name">
+          APPLE A1419 iMac 27" Retina <br>
+          5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
+
+      <div class="card__votes">
+        <div class="card__stars stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price_label">
+          Price:
+        </p>
+
+        <p class="card__price_value">
+          $2,199
+        </p>
+      </div>
+
+      <a
+        class="card__button_buy"
```

Comment:
This naming is wrong, classnames in html should be written in kebab-case, so it should be `card__button-buy` instead

Reply:
It should be changed in other classes as well, for example `card__price_label` is also wrong

Reply:
updated in 'solution2'